cluster/gce/util.sh

```diff
@@ -1265,8 +1265,12 @@ function test-setup {
   until gcloud compute firewall-rules describe --project "${PROJECT}" "${NODE_TAG}-${INSTANCE_PREFIX}-http-alt" 2> /dev/null || [ $(($start + 1200)) -lt `date +%s` ]
   do sleep 5
   done
-  # Check if the firewall rule exists and fail if it does not.
-  gcloud compute firewall-rules describe --project "${PROJECT}" "${NODE_TAG}-${INSTANCE_PREFIX}-http-alt"
+  # Note, determine if we timed out, or if we found the firewall, you might think we could just
+  # check with gcloud, but sometimes that call flakes.  https://github.com/kubernetes/kubernetes/issues/20306
+  if [ $(($start + 1200)) -lt `date +%s` ]; then
+    echo "Failed to create a firewall ${NODE_TAG}-${INSTANCE_PREFIX}-http-alt in ${PROJECT}.
```

Comment:
note this is missing the final ".

should also probably use `${color_red}` and echo to stderr.